datm/cime_config/stream_definition_datm.xml

```diff
@@ -236,7 +236,6 @@
       <file first_year="$DATM_YR_START" last_year="$DATM_YR_END">$DIN_LOC_ROOT/atm/cdeps/v1/$NEONSITE/%ym.nc</file>
```

Comment:
We only have a 'v1' on /glade/p/cesmdata/cseg/inputdata/atm/cdeps. However, should both of the NEON streams (NEON and NEON.PRECIP) be pulling in 'v2' or 'v3' data from the NEON side?

Reply:
@jedwards4b after discussing with @wwieder , it sounds like this gets overwritten to the latest version elsewhere, but should we update to 'v2'?

Reply:
Just to clarify, we want the feature to continue drawing from the latest version if no version is specified (e.g. some NEON sites have v3 data now!).  

I get confused because the run_neon script has a feature that allows users to specify what version is being used, but the cdeps code seems hard coded to v1.

Reply:
Okay. I agree that logically makes sense, but I'm not really sure where this actually gets updated, and I think we want to take the latest version (v3 is coming online soon/partially available for one site). For the sake of getting this PR merged and maintaining flexibility, I think it may make the most sense to add a comment to clarify this in CDEPS, and potentially tweak run_neon in the future.